### DISCUSSION TITLE
Nice! Can be used to host blog?

### DISCUSSION THREAD

**Discussion by speg** [simonw/datasette#1307] [General] [answered]
Aaahhh. I was just about to build a little HTTP server that pulled in some static pages and sprinkle some SQLite around. Is this tool already that? 😿 

PS - Love the basic yet consistent UI! No CSS framework it looks like. Awesome work!

**Comment by simonw** [comment, marked as answer]
It certainly can! I'm running three blog-like sites on Datasette at the moment:

- https://datasette.io - code at https://github.com/simonw/datasette.io
- https://til.simonwillison.net/ - code at https://github.com/simonw/til
https://www.niche-museums.com/ - code at https://github.com/simonw/museums
